configs/llm-360-amber-s3.yaml

```diff
@@ -0,0 +1,508 @@
+run_name: llm-360-amber-data-hyper-init-repro-3
+seed: 6198
+dry_run: false
+
+wandb:
+  name: ${run_name}
+  project: olmo-medium
+  group: ${run_name}
+  tags:
+    - epwalsh-amberish-stability
+
+model:
+  d_model: 4096
+  n_heads: 32
+  n_layers: 32
+  # mlp_ratio: 6
+  mlp_hidden_size: 22016
+  weight_tying: false
+  alibi: false
+  rope: true
+  flash_attention: true
+  attention_dropout: 0.0
+  attention_layer_norm: false
+  include_bias: false
+  block_type: sequential
+  layer_norm_type: rms
+  layer_norm_with_affine: false
+  layer_norm_eps: 1e-6
+  bias_for_layer_norm: false
+  attention_layer_norm_with_affine: false
+  activation_type: swiglu
+  residual_dropout: 0.0
+  embedding_dropout: 0.0
+  max_sequence_length: 2048
+  vocab_size: 32000
+  embedding_size: 32000
+  eos_token_id: 2
+  pad_token_id: 2
+  init_device: meta
+  init_fn: normal
+  init_std: 0.02
+
+compile: null
+
+optimizer:
+  name: adamw
+  learning_rate: 3.0e-4
+  weight_decay: 0.1
+  eps: 1e-8
+  decay_norm_and_bias: true
+  decay_embeddings: false
+  betas:
+  - 0.9
+  - 0.95
+  metrics_log_interval: 1
+
+scheduler:
+  name: cosine_with_warmup
+  units: tokens
+  t_warmup: 9175040000
+  t_max: 1.25e12
+  alpha_f: 0.1
+  warmup_min_lr: 0.0
+
+tokenizer:
+  identifier: tokenizers/nousresearch-llama-2-7b-hf.json
+  truncate_direction: right
+
+save_folder: runs/${run_name}
+remote_save_folder: s3://ai2-llm/checkpoints/OLMo-medium/epwalsh-stability/${run_name}
+save_overwrite: true
+
+save_interval: 250
+save_interval_ephemeral: null
+save_num_checkpoints_to_keep: 2
+sharded_checkpointer: olmo_core
+
+save_interval_unsharded: null
+save_num_unsharded_checkpoints_to_keep: -1
+
+load_path: null
+
+max_duration: 2ep
+global_train_batch_size: 2240
+device_train_microbatch_size: 2
```

Comment:
I didn't change this originally, but should we try to up this since our sequence length is now half the size?

Reply:
Might be able to get away with that but my per-device batch isn't divisible by 4, I want to keep global batch size the same for now.